docs/migrations.md

```diff
@@ -96,8 +96,34 @@ To generate a new migration script, execute:
 Where `${branch_name}` is either the name of the extension's module (if the
 script is for an extension) or `haas` (if the script is for HaaS core).
 
-Alembic will generate a migration script in the appropriate directory. Again,
-sanity check the output; Alembic's guesses should not be trusted blindly.
+Alembic will look at the differences between the schema in the database and
+the one derived from the HaaS source, and generate its best attempt at a
+migration script. The script will be stored in `haas/migrations/versions`.
+
+The value of `down_revision` should be the identifier of the previous migration script.
+The value of `branch_labels` should be `('<branch_name>',)` where `branch_name` should match what was used in the command to generate the migration script.
```

Comment:
Please hard-wrap paragraphs to a maximum of 79 chars; without that it's annoying to read in a terminal.